hack/test-go.sh

```diff
@@ -1,7 +1,19 @@
 #!/usr/bin/env bash
 # single test: go test -v ./pkg/storage/
 # without cache: go test -count=1 -v ./pkg/storage/
-set -e -x
+set -eo -x pipefail
+
+while true; do
+  case "$1" in
+    -u|--use-static-check)
```

Comment:
consider flipping this logic to `--skip-static-check` and then the default `make test`  targetincludes the static check, but this option skips it